media-libs/sdl_rtf/sdl_rtf-0.1.0.recipe

```diff
@@ -1,13 +1,54 @@
-DESCRIPTION="SDL-rtf"
+SUMMARY="Rich Text Format decoding add-on for SDL"
+DESCRIPTION="This is a sample library which allows you to display Rich Text Format (RTF) \
+documents in your SDL applications. It comes with an example program \
+"showrtf" which displays an RTF file using an SDL_ttf font engine."
 HOMEPAGE="http://www.libsdl.org/projects/SDL_rtf"
+COPYRIGHT="2003-2009 Sam Lantinga"
+LICENSE="GNU LGPL v2"
+REVISION="2"
 SOURCE_URI="http://www.libsdl.org/projects/SDL_rtf/release/SDL_rtf-0.1.0.tar.gz"
-CHECKSUM_MD5="fe36733167b5c89f128414f32612121a"
-REVISION="1"
-STATUS_HAIKU="stable"
-DEPEND="media-libs/sdl-ttf >= 2.0.9"
+CHECKSUM_SHA256="3dc0274b666e28010908ced24844ca7d279e07b66f673c990d530d4ea94b757e"
+SOURCE_DIR="SDL_rtf-$portVersion"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	sdl_rtf$secondaryArchSuffix = $portVersion
```

Comment:
The same as loadpng: if nothing depends on sdl_rtf I would consider dropping the recipe.

Reply:
sdl_rtf probably isn't used by anything.  It was a sample proof of concept library from the sdl developers: https://www.libsdl.org/projects/SDL_rtf/
Says it comes with an example program "showrtf", perhaps this should be added to the PROVIDES?